deploy/crds/kieapp.crd.yaml

```diff
@@ -522,6 +522,16 @@ spec:
                         description: The image tag to use e.g. 7.9.0, this param is
                           optional for custom image.
                         type: string
+                      jbpmCluster:
+                        description: Enable the clustering between Kieservers and
+                          add one replicas if only one is configured
+                        properties:
+                          enableCluster:
+                            type: boolean
```

Comment:
It would be good to also add description for this property. It can be helpful in KieApp yaml file editor in ocp

Reply:
ok